core/src/main/scala/stryker4s/mutants/applymutants/MatchBuilder.scala

```diff
@@ -20,6 +21,16 @@ class MatchBuilder {
         rest transformOnce {
           case found if found.isEqual(origStatement) && found.pos == origStatement.pos =>
             buildMatch(mutant)
+        } match {
+          case Success(value) => value
+          case Failure(exception) =>
+            error(
+              s"Failed to construct pattern match: original statement [$origStatement] " +
+                s"on line [${origStatement.pos.endLine}] for file [${origStatement.pos.input}].)")
+            error(s"Failed mutation(s) ${mutant.mutantStatements.mkString(",")}.")
+            debug(exception.getMessage)
+
+            throw exception
```

Comment:
The original exception gives a *lot* of confusing logging. Maybe we should throw a custom exception here, indicating what happened